org.srb2.SRB2Kart.metainfo.xml

```diff
@@ -50,52 +50,7 @@
-          <li>Up to 126 servers may now appear in the server list</li>
-          <li>Caret color codes may be used in server names</li>
-          <li>Players may now change their name no more than five times per minute while playing online</li>
-        </ul>
-        <p>Lua scripting changes</p>
-        <ul>
-          <li>Use player.ping to access the player's latency in milliseconds</li>
-          <li>Fixed displayplayers.iterate crashing in replays</li>
-          <li>Added replayplayback, which tells whether you are in a replay</li>
-          <li>Added file I/O support</li>
-          <li>Fixed sound volume not working on sounds that have an origin</li>
-        </ul>
-        <p>OpenGL changes</p>
-        <ul>
-          <li>Fixed flat rotation being slightly off</li>
-          <li>Fixed the game sometimes crashing on startup due to ogllog.txt</li>
-          <li>Fixed encore colormaps being slightly off</li>
-        </ul>
-        <p>Miscellaneous changes</p>
-        <ul>
-          <li>Added support for Discord Rich Presence. Some details of the game, such as what map you're playing, or how many players are on a server will be displayed on your profile. You may also invite others to join your game. This works even if you are not the server host, but just an ordinary player! Others may ask to join you too. You'll receive a notification in game if they do, and you can choose to accept or deny it.</li>
-          <li>Adjusted item odds in Battle mode. Orbinauts, Jawz, Invincibility and Grow are far more likely to appear. Bananas and Sneakers much less. Also removed item odds scaling with bumper count. Now everyone is on an equal playing field.</li>
-          <li>The last used renderer is now saved. This means you only need to start the game with OpenGL once and the game will remember it thereafter.</li>
-          <li>Fixed a minor bug with aliases in the console</li>
-          <li>Fixed first person camera breaking in replays</li>
-          <li>Fixed clearbans not clearing ban.txt</li>
-          <li>Fixed Record Attack ghosts potentially crashing if recorded with Lua added</li>
-          <li>SPB is no longer forced to appear in a 1v1</li>
-          <li>Fixed Thunder Shield item odds</li>
-          <li>Fixed a memory leak with pk3 files</li>
-          <li>Fixed SPB, Shrink and Hyudoro items appearing only once per race</li>
-          <li>Made looking back instantaneous</li>
-          <li>Fixed Toad</li>
-          <li>Fixed a memory leak if replay is not saved</li>
-          <li>You can no longer "line ride" offroad</li>
-          <li>Increased rarity of a previously played map showing up again in vote</li>
-          <li>Added join, leave and kick sound effects</li>
-          <li>Updated bonuschars with brand new Sakura sprites!</li>
-        </ul>
+        <p>The release notes can be found at <code>https://github.com/STJr/Kart-Public/releases/tag/v1.3</code>.</p>
```

Comment:
seems the builder doesn't like that dot at the end, should be okay with removing it.